joern-cli/frontends/jimple2cpg/src/main/scala/io/joern/jimple2cpg/astcreation/AstCreator.scala

```diff
@@ -287,6 +287,7 @@ implicit class JvmStringOpts(s: String) {
     * <code>I</code> to <code>int</code>.
     * @return
     */
-  def parseAsJavaType: String = Type.getType(s).getClassName.replaceAll("/", ".")
+  def parseAsJavaType: String =
+    Try { Type.getType(s).getClassName.replaceAll("/", ".") }.getOrElse("unknownType")
```

Comment:
Perhaps use `io.joern.x2cpg.Defines` here
```suggestion
    Try { Type.getType(s).getClassName.replaceAll("/", ".") }.getOrElse(Defines.Any)
```